proversity_reports_script/report_backend/activity_completion_report.py

```diff
@@ -67,6 +70,11 @@ def create_csv_file(self, file_name, body_dict, headers, *args, **kwargs):
                 writer.writerow(row)
 
         self.upload_file_to_storage(file_name, path_file)
+        update_sheets_data(
```

Comment:
This change will BREAK the Capaciti report job, as the Capaciti report does not update any Spreasheets. So in order to keep some backward compatibility, I propose, that EXTRA_DATA contain a feature flag, whether to update the Spreadsheet or not. @cocococosti

Reply:
you're right, thank you! I will add a feature flag for this @Squirrel18

Reply:
I just checked the configs in the server and capaciti doesn't have the activity completion report, I will add a condition here anyways in case other clients do use it.